_inc/client/plans/single-product-backup.scss

```diff
@@ -89,11 +93,23 @@ input[type='radio'].plan-radio-button__input {
 .single-product-backup__radio-buttons-container {
 	display: flex;
 	flex-flow: column nowrap;
+	margin-bottom: 21px;
 
 	@include breakpoint( ">660px" ) {
 		flex-flow: row wrap;
 		justify-content: space-around; // IE11 fallback.
 		justify-content: space-evenly;
+		position: relative;
+
+		&::before {
+			display: block;
+			border-left: 1px solid $gray-text-min;
```

Comment:
It feels that the border is too dark compared to mockups. Could we maybe use here some a lighter grey swatch?

Reply:
Switched it up in 9a8cc1dc783be55e20acb1ac58c2ddd824b89ef7, think it looks much better now!